tests/testthat/test-runner.R

```diff
@@ -323,15 +323,13 @@ test_that("check_timeout kills timed out reports", {
   path <- orderly_git_example("demo")
   runner <- orderly_runner(path, workers = 2)
 
-  key1 <- runner$submit_task_report("minimal", timeout = 0)
-  key2 <- runner$submit_task_report("slow10", timeout = 0)
-  ## Sleep for a bit to allow "minimal" report to post TASK_COMPLETE message
-  Sys.sleep(5)
+  key <- runner$submit_task_report("slow10", timeout = 0)
+  Sys.sleep(2) ## Sleep to let runner pick up tasks
   msg <- capture_messages(killed <- runner$check_timeout())
-  task2 <- get_task_id_key(runner, key2)
-  expect_equal(killed, task2)
-  expect_equal(msg, sprintf("Successfully killed '%s', exceeded timeout of 0\n",
-                            task2))
+  task <- get_task_id_key(runner, key)
```

Comment:
This was failing under covr - not code change which would have triggered this so probably just being a bit flaky. I've tried to simplify it to reduce changes of intermittent failures